logic/auth.js

```diff
@@ -195,7 +198,7 @@ async function seed(user) {
     try {
         const { seed } = await diskLogic.readUserFile();
 
-        const decryptedSeed = await iocane.createSession().decrypt(seed, user.plainTextPassword);
+        const decryptedSeed = await iocane.createSession().decrypt(seed, Buffer.from(user.plainTextPassword, "base64").toString("utf-8"));
```

Comment:
Have you tested this? Looks like `user.plainTextPassword` is already decoded in the change above so this may not be needed.

If it is needed then it's confusing to have a variable called `plainTextPassword` when the value is not plain text but is Base64 encoded. It should be `base64EncodedPassword`.

But if we can decode as soon as the route is hit then none of these changes are needed.

Reply:
Yes I've tested it and had no errors but you're right its not normal I will check it now!

Reply:
You were clearly right I've just patched it in fc5d2ca